### PR TITLE
[EASI-4361] Export options, linking table translation

### PR DESCRIPTION
**Pull request by patrickseguraoddball**
# EASI-4361

## Changes and Description

The fields in the new translations are just placeholders!  Feel free to add/edit/remove at will!

- Added optional exportOptions translation type for restricted field
- Added two new translation tables for linking tables
  - plan_document_solution_link
  - existing_model_link
- Export

## How to test this change

Verify export and types look correct

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: pkg/graph/schema/types/existing_model_link_translation.graphql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"""
+Represents exsiting model link translation data
+"""
+
+type ExistingModelLinkTranslation {
+    exsitingModelID: TranslationField! @goTag(key: "db", value: "existing_model_id")
```
Comment (StevenWadeOddball) [inline pkg/graph/schema/types/existing_model_link_translation.graphql:6]: there is a slight type for existingModelId and the comment

File: src/i18n/en-US/modelPlan/existingModelLink.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { TranslationExistingModelLink } from 'types/translation';
+
+import {
+  TranslationDataType,
+  TranslationFormType
+} from '../../../gql/gen/graphql';
+
+export const existingModelLink: TranslationExistingModelLink = {
+  exsitingModelID: {
```
Comment (StevenWadeOddball) [inline src/i18n/en-US/modelPlan/existingModelLink.ts:9]: existing has a typo here and on the field definitions.

**Review by StevenWadeOddball — COMMENTED** [review]
Thanks Patrick! This is looking good.

I think we need to add more fields to the translation for the links. I can also update it when I build the backend queries if better!

Existing Model links should have
1. existing_model_id
2. current_model_plan_id
3. field_name

DocumentSolutionLink should have
1. solution_id
2. document_id

**Comment by patrickseguraoddball** [comment]
Yep!  I think I left a note in th edscription of the PR that it's just a placeholder field, feel free to modify!

**Pull request merged**